va/va.go

```diff
@@ -151,11 +155,12 @@ func New(
 	return va
 }
 
-func (va VAImpl) ValidateChallenge(ident string, chal *core.Challenge, acct *core.Account) {
+func (va VAImpl) ValidateChallenge(ident string, chal *core.Challenge, acct *core.Account, identType string) {
```

Comment:
If the `acme.Identifier` type I mentioned above is added it could be used to replace the two `ident` and `identType` arguments with one `acme.Identifier` argument to simplify this function signature a little bit.